### ISSUE TITLE
Support Behat 3.0

### ISSUE THREAD

**Issue by stof** [mprzytulski/phpstorm-behat#36]
Supporting Behat 3.0 involves a few changes:
- the ContextInterface has been renamed to Context, which is probably why the special icon is not used on the file
- steps can be defined using the Turnip syntax rather than regexes (and it is even the default for the snippet generation). This needs to be supported

**Comment by onchainguy-btc** [comment]
+1

**Comment by stof** [comment]
@seb0zz note that PhpStorm 8 (currently in EAP) features some Behat integration.

Last time I tried this plugin, it was totally broken with PhpStorm 7

**Comment by mikelohmann** [comment]
I am currently using Storm 8 EAP with behat plugin. Sometimes the step definitions don’t lead to a method, but normally it works well. It is the first time since some month the plugin works again for me.
Am 05.08.2014 um 16:12 schrieb Christophe Coevoet notifications@github.com:

**Comment by stof** [comment]
@mikelohmann the Behat support in PhpStorm 8 EAP is _not_ this repo. This repo is an abandonned attempt to provide the same feature as a third-party plugin

**Comment by mikelohmann** [comment]
I just wanted to support your answer by some experience with that, because I tried this repos plugin several times in different versions of storm and failed. The only thing worked for me was the plugin of the Storm 8 EAP.

Am 05.08.2014 um 20:16 schrieb Christophe Coevoet notifications@github.com:

**Comment by onchainguy-btc** [comment]
Yeah the basic support in PHPStorm 8 EAP works for me, too. Looks like a good alternative to the plugin as the plugin has not been very stable recently.
